app/controllers/contributions_controller.rb

```diff
@@ -18,6 +18,7 @@ def index
     if current_user_can?(:manage, Contribution)
       # show all contributions and events
       @events = EDM::Event.where({})
+      @delete_buttons = true
```

Comment:
Could we name this something else? The controller should not need to know whether or not buttons are to be displayed by the view. Alternatively move it into a helper or presenter concern method.